translations/am.json

```diff
@@ -23,6 +23,7 @@
   "alertRole": "እርምጃ ሚና",
   "all": "ሁሉም",
   "approve": "ተግባር",
+  "batch": "Batch No",
```

Comment:
We should be updating this file.

Reply:
Yah, I will update all the other translation files, thanks @donaldkibet

Reply:
Sorry we should not be updating the translation files. Only update the `en.json` file

Reply:
Remind me again, why shouldn't we update the other files? I am not sure how that works but I like it because I spend a lot of time on those translations.